packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -574,6 +574,25 @@ const Conversation = SparkPlugin.extend({
   },
 
   /**
+   * Sets announcement mode for convo
+   * @param {Object} conversation
+   * @param {ShareActivity|Array<File>} activity
+   * @returns {Promise<Activity>}
+   */
+  setAnnouncement(conversation, activity) {
```

Comment:
Let's see about generalizing this a bit. I'm assuming there will be additional spaceProperties to set at some future point (though this is atleast the second verb we've added for setting tags so I have little hope will stick to this interface). 

Take a look at the bottom of conversation.js where we generate a bunch of functions because they're identical except for verb.